tests/js/server/aql/aql-ternary.js

```diff
@@ -246,7 +255,7 @@ function ahuacatlTernaryCollectionTestSuite () {
       db._drop(cn);
       c = db._create(cn);
       for (var i = 0; i < 100; ++i) {
-        c.insert({value:i});
+        c.insert({_key: "test" + i, value: i});
```

Comment:
please make this a bulk insert

Reply:
It just 100 documents inserted in a single test case... Anyway, I did it.